src/index.html

```diff
@@ -5,9 +5,57 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+
+    <div class="card" data-qa="card">
+      <div class="card__content">
+
+        <img
+          src="/images/imac.jpeg"
+          alt="iMac"
+          class="card__image"
+        >
+
+        <h1 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+
+        <h2 class="card__productCode">
+          Product code: 195434
+        </h2>
+
+        <div class="card__reviews">
+          <div class="card__stars card__stars--4">
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+          </div>
+
+          <p class="card__number">
+            Reviews: 5
+          </p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__text">Price:</p>
+          <p class="card__sum">$2,199</p>
+        </div>
+
+        <div class="card__buttom">
+          <a
+          href="#"
+          class="card__link"
+          data-qa="hover"
+          >Buy</a>
+        </div>
```

Comment:
Redudant div wrapper. Pay attention on code style
```suggestion
        <a
          href="#"
          class="card__link"
          data-qa="hover"
        >
          Buy
        </a>
```

Reply:
✅